pkg/objectstore/s3/s3.go

```diff
@@ -123,9 +128,13 @@ func GetObjLockInfo(backupLocation *stork_api.BackupLocation) (*common.ObjLockIn
 			objLockInfo.RetentionPeriodDays = aws.Int64Value(out.ObjectLockConfiguration.Rule.DefaultRetention.Days)
 		} else {
 			//This is an invalid object-lock config, no default-retention but object-loc enabled
+			logrus.Errorf("GetObjLockInfo: invalid config: object lock is enabled but default retention period is not set on the bucket")
 			objLockInfo.LockEnabled = false
 			return nil, fmt.Errorf("invalid config: object lock is enabled but default retention period is not set on the bucket")
 		}
 	}
+	// This debug statement will not command as both err and out can be nil.
```

Comment:
nit: please fix any typo in the comment.  "command" sounds wrong

Reply:
Fixed